packages/mds-policy-service/service/validators.ts

```diff
@@ -139,3 +139,12 @@ export const { validate: validatePolicyMetadataDomainModel, isValid: isValidPoli
       policy_metadata: { type: 'object' }
     }
   })
+
+export const { validate: validatePresentationOptions, isValid: isValidPresentationOptions } =
+  SchemaValidator<PresentationOptions>({
+    $id: 'PresentationOptions',
+    type: 'object',
+    properties: {
+      withStatus: { type: 'boolean' }
+    }
+  })
```

Comment:
Why isn't status just part of the domain model again? Is it because it wasn't part of the original spec? Could we just have a domain model that `extends PolicyDomainModel`?

Reply:
It's not part of the original spec, and it's mutable which could make it _appear_ like the PolicyJSON is being mutated when it really isn't. 

We could have a domain model that extends it, but I think that'd be a bit more friction heavy unless we wanted to add entirely new service methods for reading policies with status.

Reply:
& Also status is partially derived from what time it is now, so better as a computed field. (?)